.github/ISSUE_TEMPLATE/bug_report.yaml

```
@@ -0,0 +1,57 @@
+name: Bug Report
+description:  Create a report to help us improve
+labels: ["Type: Bug", "Needs: Triage", "Needs: Lead"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **IMPORTANT: Before posting, remove sensitive data, such as passwords, secret keys, session cookies, etc.**
+
+        Thanks for taking the time to fill out this bug report! 👍
+  - type: textarea
+    attributes:
+      label: Problem
+      description: A short description of the issue you're facing.
+      value: |
+        #### Describe the bug
+
+        #### Evidence / Screenshot
+
+        #### Relevant URL
+
+  - type: textarea
+    attributes:
+      label: Reproducing the bug
+      description: What steps caused you to find the bug?
+      value: |
+        #### Steps to reproduce
+        1. Go to ...
+        2. Do ...
+
+        #### Expected behavior
+
+        #### Actual behavior
```

These can be a bit more condensed. The final issue is a little spread out making it hard to read for me. Also thanks for flipping the order on these, that's been bothering me for eons :P 

```suggestion
        * Expected behavior: 
        * Actual behavior: 
```